clib/cList.ml

```diff
@@ -160,20 +160,31 @@ let assign l n e =
 
 (** {6 Filtering} *)
 
-let rec filter_loop f p = function
-  | [] -> ()
-  | x :: l' as l ->
-    let b = f x in
-    filter_loop f p l';
-    if b then if p.tail == l' then p.tail <- l else p.tail <- x :: p.tail
+(* [filter_loop f (Some (c0,l0)) c l] will do c0.tail <- l0 if [for_all f l] *)
```

Comment:
Not sure what you did (cherry pick?) but this shouldn't be appearing in the change list

Reply:
I'm confused, too. I rebased from master and it told me, I was 1 commit ahead from origin/extraction_extension. So I pushed. I think that was wrong. I'll look at it.

Reply:
Okay, resolved that. Seems that my local master was not really in sync with my origin/master.